IDR/notebook/scripts/nbconverted/1.describe_screens.py

```diff
@@ -137,17 +168,71 @@ def describe_screen(screen_id):
                     "Channels"
                 ]
 
-                # Clean channels value and separate into stain and stain target
-                stain = set()
-                stain_target = set()
-                for entry in channels.split(";"):
-                    temp_list = entry.split(":")
-                    stain.add(temp_list[0])
-                    stain_target.add(temp_list[1])
+                # Clean channels value and add to stain:target list
+                stains_targets = list()
+                for channel in channels.split(";"):
+
+                    # For channel entries with 'stain:target' format
+                    if bool(re.search(r"([\:])+", channel)):
+                        # Split stain and target for whitespace trimming
+                        split = channel.split(":")
+
+                        # Trim whitespace
+                        stripped = [s.strip() for s in split]
```

Comment:
can this, the previous, and the following logic be combined and simplified?

```python
stain, target = [s.strip() for s in channel.split(":")
```

This feels more pythonic to me 🐍

Reply:
@gwaybio The line was changed to your suggestion and the following lines assigning stain and target from the list indices were removed.

Added in f94bfe0fb0f25d22e9d8eb21eefa334f44836fab.